packages/pds/src/api/com/atproto/repo/createRecord.ts

```diff
@@ -44,6 +46,16 @@ export default function (server: Server, ctx: AppContext) {
           'Unvalidated writes are not yet supported.',
         )
       }
+      if (AppBskyFeedPost.isRecord(record)) {
+        if (record.tags) {
+          for (const tag of record.tags) {
+            if (!isValidHashtag(tag)) {
+              throw new InvalidRequestError(`Invalid hashtag: ${tag}`)
+            }
```

Comment:
Checks for slurs here

Reply:
we do this check over in `src/repo/prepare.ts` - would be good to keep it all in one spot 👌 

also would be good to mirror the check over in the auto-moderator code in the appview

Reply:
Done!